qiskit_metal/renderers/renderer_base/renderer_base.py

```diff
@@ -377,7 +377,7 @@ def get_unique_component_ids(
         that the name of component exists in QDesign. If QDesign doesn't
         contain any component, or if all components in QDesign are found in
         highlight_qcomponents, return an empty list; otherwise return a list of
-        unique components to be sent to Ansys. The second returned item, an
+        unique components to be sent to the renderer. The second returned item, an
```

Comment:
In the Args part of doc-string.  There is a reference to Ansys.

Reply:
Removed Ansys reference.

Reply:
Look below in the Args,it is still there.

`highlight_qcomponents (Union[list, None], optional): Components to render into **Ansys**. Defaults to None.`

Reply:
<img width="754" alt="Screen Shot 2021-04-14 at 2 20 08 PM" src="https://user-images.githubusercontent.com/16978018/114780704-9a6eb300-9d2c-11eb-96e8-7cbdd7d4a625.png">

Reply:
No it's not.

Reply:
Agreed.  It is fixed.

Reply:
Thank you!